2017-11-18 17:37:19  [ main:0 ] - [ ERROR ]  {dataSource-1} init error
java.sql.SQLException: com.mysql.jdbc.Driver
	at com.alibaba.druid.util.JdbcUtils.createDriver(JdbcUtils.java:570)
	at com.alibaba.druid.pool.DruidDataSource.init(DruidDataSource.java:697)
	at com.alibaba.druid.pool.DruidDataSource.getConnection(DruidDataSource.java:1058)
	at com.alibaba.druid.pool.DruidDataSource.getConnection(DruidDataSource.java:1054)
	at com.stemcloud.liye.project.common.DbConnectionPool.getConnection(DbConnectionPool.java:42)
	at com.stemcloud.liye.project.common.CrudTest.main(CrudTest.java:20)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at com.intellij.rt.execution.application.AppMain.main(AppMain.java:144)
Caused by: java.lang.ClassNotFoundException: com.mysql.jdbc.Driver
	at java.net.URLClassLoader.findClass(URLClassLoader.java:381)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:424)
	at sun.misc.Launcher$AppClassLoader.loadClass(Launcher.java:331)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:357)
	at java.lang.Class.forName0(Native Method)
	at java.lang.Class.forName(Class.java:264)
	at com.alibaba.druid.util.JdbcUtils.createDriver(JdbcUtils.java:568)
	... 10 more
2017-11-18 17:38:42  [ main:0 ] - [ INFO ]  com.mysql.jdbc.Driver
2017-11-18 17:38:42  [ main:553 ] - [ INFO ]  {dataSource-1} inited
2017-11-18 17:42:52  [ main:0 ] - [ INFO ]  {dataSource-1} inited
